lib/cockpit-components-listing-panel.scss

```diff
@@ -1,41 +1,41 @@
 .ct-listing-panel {
   &-head {
-    background: linear-gradient(to top, 
+    background: linear-gradient(to top,
       var(--pf-global--BorderColor--100) 0,
       var(--pf-global--BorderColor--100) 1px,
       transparent 1px
     );
     display: flex;
-    align-items: flex-end;
+    align-items: end;
```

Comment:
This introduces a difference to cockpit's master, which also has  "flex-end". See https://github.com/cockpit-project/cockpit/pull/14360 . Can you please sync this to current master?

Reply:
Ups good catch, I just copied the files as were on my branch which was not rebased.